boards/nxp/frdm_mcxn947/board.c

```diff
@@ -285,6 +285,13 @@ static int frdm_mcxn947_init(void)
 	CLOCK_AttachClk(kFRO_HF_to_ADC0);
 #endif
 
+#if DT_NODE_HAS_STATUS_OKAY(DT_NODELABEL(usb0)) && (CONFIG_USB_KINETIS || CONFIG_UDC_KINETIS)
+	CLOCK_AttachClk(kCLK_48M_to_USB0);
+	CLOCK_EnableClock(kCLOCK_Usb0Ram);
+	CLOCK_EnableClock(kCLOCK_Usb0Fs);
+	CLOCK_EnableUsbfsClock();
+#endif
```

Comment:
How was this tested?

Reply:
WIth a custom board that I made...
You're right, on the FRDM board the USB0 pins are only connected to two TP. Therefore it is not pertinent to put this configuration in the `board.c` file.

But why all these configurations are not done in a `soc.c` file, "at the soc level"?

Reply:
We are moving to making the clock configuration board specific as these should be managed at a board level. Kindly delete this from board.c

Reply:
It is a bit a shame because it means that if someone wants to create a custom board based on this soc, he necessarily needs to do some research how to use the HAL.
But I also understand the intention to manage it at the board level, I will remove this modification.